packages/open-next/src/adapters/revalidate.ts

```diff
@@ -0,0 +1,51 @@
+import { SQSEvent } from "aws-lambda";
+import fs from "node:fs";
+import path from "node:path";
+import { request } from "node:https";
+import { IncomingMessage } from "node:http";
+import { debug } from "./logger.js";
+
+interface PrerenderManifest {
+  preview: {
+    previewModeId: string;
+    previewModeSigningKey: string;
+    previewModeEncryptionKey: string;
+  };
+}
+
+function loadPrerenderManifest() {
+  const filePath = path.join("prerender-manifest.json");
+  const json = fs.readFileSync(filePath, "utf-8");
+  return JSON.parse(json) as PrerenderManifest;
+}
+
+export const handler = async (event: SQSEvent) => {
+  const { preview } = loadPrerenderManifest();
+  for (const record of event.Records) {
+    const body = record.body;
```

Comment:
Do you not need to call `cloudfront.createInvalidation` to invalidate the path?

Reply:
For ISR that's not necessary, for on demand revalidation we could but it's not handled here but directly in users code.
Should we do something in open-next, i'm not entirely sure, cloudfront invalidation are slow and pretty expensive. After the 1000 first invalidation, it's 0.005$ per invalidation.